include-build/roborazzi-core/src/commonMain/kotlin/com/github/takahirom/roborazzi/KotlinxIo.kt

```diff
@@ -0,0 +1,58 @@
+package com.github.takahirom.roborazzi
+
+import kotlinx.io.buffered
+import kotlinx.io.bytestring.encodeToByteString
+import kotlinx.io.files.Path
+import kotlinx.io.files.SystemFileSystem
+import kotlinx.io.readString
+import kotlinx.io.write
+
+val Path.absolutePath: String
+  get() = this.toString()
+
+val Path.nameWithoutExtension: String
+  get() = name.substringBeforeLast(".")
+
+fun Path.relativeTo(base: Path): Path {
```

Comment:
Looks great! When I tried this, I wasn't able to find a way to implement it. Could we add a small unit test for this?

Reply:
Good call. I'll add unit test for it.

Reply:
Added with 51e989a